src/app/app.routes.ts

```diff
@@ -16,7 +16,6 @@ export const routes: Routes = [
   {
     path: 'checkout',
     loadChildren: './checkout/checkout.module#CheckoutModule',
-    canActivate: [ CanActivateViaAuthGuard ],
```

Comment:
any reason fir removing this ?

Reply:
This is not useful for the checkout process. when user application redirected after payubiz payment. it forces the user to login.